src/views/LegalView/LegalView.style.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import { Text } from '@/shared/components'
+import { colors, sizes } from '@/shared/theme'
+import styled from '@emotion/styled'
+
+export const Box = styled.div`
+  scroll-behavior: smooth;
+  text-align: left;
+  margin-bottom: ${sizes(10)};
+  position: relative;
+  height: 85vh;
+  width: 100%;
+  background-color: ${colors.gray[800]};
+  overflow: auto;
+`
+export const TextWrapper = styled.div`
+  margin: ${sizes(9)} ${sizes(17)} ${sizes(9)} ${sizes(8)};
+  a {
+    text-decoration: none;
+    color: ${colors.gray[50]};
+  }
+`
+const _Paragraph = styled(Text)`
+  margin-top: ${sizes(6)};
+  ol + & {
+    margin-top: 0;
+  }
+`
+
+export const Paragraph: React.FC<{ header?: boolean }> = ({ header, ...props }) => (
```

Comment:
We need to adjust colors for the text. The body text is too dark for the background, the contrast is too low. In terms step we use `secondary` for everything except the header